assets/js/common/ComposedFilter/ComposedFilter.jsx

```diff
@@ -1,10 +1,22 @@
 import React, { useEffect, useState } from 'react';
+import { EOS_SEARCH } from 'eos-icons-react';
 import Button from '@common/Button';
+import Input from '@common/Input';
 import Filter from '@common/Filter';
 import DateFilter from '@common/DateFilter';
 
 const renderFilter = (key, { type, ...filterProps }, value, onChange) => {
   switch (type) {
+    case 'search_box':
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** brilliant, I like it, it's effective‏